contracts/interfaces/0.8.x/IGenArtDependencyConsumer.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+/**
+ * @notice Interface for Art Blocks contracts that expose script details
+ */
+interface IGenArtDependencyConsumer {
```

Comment:
Oops, probably would have been smart if these functions were included in the `IGenArt721CoreContractV3`.

To ensure future derivatives of the V3 core contract remain compatible with this registry, WDYT about just adding these functions to `IGenArt721CoreContractV3.sol`, and avoiding this new interface file altogether?

Reply:
Sounds good!